docs/products/access/api-reference/config-api/identity-provider-api.md

```diff
@@ -93,13 +94,104 @@ Pragma: no-cache
+
+This creates an Identity Provider from scratch
+
+* Endpoint: `/api/v1/configuration/idps`
+* Method: POST
+
+Supported Identity Provider types: `TULIP`, `OAUTH`
+
+JSON body parameters:
+
+| Param                        | Idp type     | Required | Description                                                                                                                                                                                                             |
+|------------------------------|--------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id                           | all          | yes      | Unique identifier for an Identity Provider.                                                                                                                                                                             |
+| name                         | all          | yes      | Unique name of an Identity Provider.                                                                                                                                                                                    |
+| type                         | all          | yes      | Identity Provider type. <br/> Supported types: `TULIP`, `OAUTH`.                                                                                                                                                        |
+| enabled                      | all          | no       | Specify whether an Identity Provider is enabled. <br/> Default value: `true`.                                                                                                                                           |
+| default                      | all          | no       | Specify whether an Identity Provider is default. <br/> Default value: `false`.                                                                                                                                          |
+| issuer_uri                   | TULIP        | yes      | Uri of the issuer. This URI will be used to read the OpenID Connect configuration.                                                                                                                                      |
+| client_id                    | TULIP, OAUTH | yes      | Client identifier.                                                                                                                                                                                                      |
+| secret                       | TULIP, OAUTH | depends  | Client secret. <br/> Required if client authentication method is `client_secret_basic` or `client_secret_post`                                                                                                          |
```

Comment:
nit; For consistency we could rename it to `client_secret`.

Reply:
sure! done